### DISCUSSION TITLE
Filter new, close dropdowns

### DISCUSSION THREAD

**Discussion by Illodrus** [ScratchAddons/ScratchAddons#1207] [Ideas]
unnecessary -> First suggestion: in the All/Recommended/Beta tabs in Settings, add a tab to sort by New tag

Second suggestion: When the Scratch Addons Settings page is first opened:
have all dropdowns be closed
OR
have a checkbox so you can toggle: dropdowns open when settings is opened/not open when settings is opened
(reason: for those who commonly open settings, it is easier to read and look at closed dropdowns and it is difficult or annoying to manually close them all. small quality-of-life improvement)

**Comment by cobaltt7** [comment]
Please make two issues

For the second tho, see #1200 I already did this but it was closed
________________________________

**Comment by WorldLanguages** [comment]
Keep in mind "new" addons are always at the top

**Reply by Illodrus** [reply]
Okay! I'll count this post as only the second suggestion in that case

**Reply by WorldLanguages** [reply]
Moved to #1211
